CONTRIBUTING.md

```diff
@@ -1,10 +1,11 @@
 # Contributing
 
 For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+In particular, before contributing please review our [contributor licensing guide](https://github.com/cyberark/community/blob/master/CONTRIBUTING.md#when-the-repo-does-not-include-the-cla)
```

Comment:
The piece of the URL after the `#` that specifies a header doesn't seem to do anything. I'm guessing it refers to a header that got removed? We should probably just link to the file without any header specified since they're more prone to changing than the file itself.

Reply:
it's actually linking to a header I'm adding - that's why this PR depends on cyberark/community#74

there's a lot in that file, so I'd prefer to leave the header to point to the specific section if I can. if it helps convince you, I'm making this update across a bunch of projects to ensure links continue to work :) I'm committed to doing the same if I change the links in the future!